content/manual/os/s3.md

```diff
@@ -35,23 +35,25 @@ Finally you'll need to create an API key so you can configure your ship to have
 
 Click API in the bottom left of the DigitalOcean navigation bar on the left side of the webpage.
 
-Generate a new token, you'll need to copy the access key and secret. Don't navigate away from this page as the secret will only be shown when it is initially created.
+Generate a new `Spaces access key`. Note that this is distinct from a `Personal access token`. You'll need to copy the access key and secret. Don't navigate away from this page as the secret will only be shown when it is initially created.
 
 Now that we have your S3 ready, we can update the information on your ship.
 
 ### Add your credentials: Landscape
 
-Navigate to your `https://<your-ship-url>/~settings#s3`, or click your sigil in the upper-right corner of the window and navigate to System Preferences -> Remote Storage.
+Navigate to your `https://<your-ship-url>/apps/grid/leap/system-preferences/storage`, or click your sigil in the upper-left corner of the window and navigate to Settings -> Remote Storage.
 
-The Endpoint depends on the region you selected, but can be found in the settings for your space shown in the image above (in that image we're using the San Francisco region).
+The Endpoint depends on the region you selected, but can be found in the settings for your space shown in the image above (in that image we're using the `sfo3` region). The Endpoint should be entered in the format `https://sfo3.digitaloceanspaces.com`.
 
-The Access Key ID and Secret Access Key is the API key we generated. Put each one in the correct field and save.
+The Access Key ID and Secret Access Key are the API keys we just generated. Put each one in the correct field and save.
 
-The S3 Bucket is the name of your digital ocean space. Once you've added that you'll also need to click "Make Active" from the drop down menu.
+The Bucket Name is the name of your digital ocean space. Once you've added that you'll also need to click "Save" at the bottom of the form.
 
 Once all of that's complete you should be good to go! Test that things are working by attempting to attach or send and image via chat. It should work. You can manage your images by logging into Digital Ocean and looking at the folder in your newly created space. You can use your same space for multiple urbit IDs by creating new API keys. Each ID will create a new directory within that space using its name by default.
 
-To manage your S3 inventory on your urbit, install Silo: web+urbitgraph://~dister-nocsyx-lassul/silo
+To manage your S3 inventory on your urbit, install Silo: web+urbitgraph://~dister-nocsyx-lassul/silo (Warning: still in Beta)
```

Comment:
silo was not working for me despite still being able to upload from Groups with same s3 config